CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 0.71.0 / 2024-01-11
+
+* [CHANGE] Configure the configuration reloader sidecar with the same web configuration as Prometheus and Alertmanager. #6194
```

Comment:
We need to have https://github.com/prometheus-operator/prometheus-operator/pull/6222/files in CHANGE as well

Reply:
Just realised this change was added recently only so won't need 🙂